trustfall_core/test_data/tests/valid_queries/fold_with_outputs_transform_and_filter_greater_than.graphql.ron

```diff
@@ -0,0 +1,19 @@
+
+TestGraphQLQuery (
+    schema_name: "numbers",
+    query: r#"
+{
+    Number(min: 30, max: 30) {
+        ... on Composite {
+            value @output
+
+            primeFactor @fold @transform(op: "count") @filter(op: ">", value: ["$min_primes"]) {
+                __typename @output
```

Comment:
Consider outputting a more interesting field than `__typename`, so that we get different results and we get a bit more testing power. For example, output `prime: value`.